.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+<!--
+
+A crisp summary
+
+Fixes #
+Ref #
+-->
+
+<!--
+Apply labels!
+-->
+
+## Checklist
+
+- [ ] Labels applied correctly? See the [contributing guidlines](../CONTRIBUTING.md).
+- [ ] Workspace compiles `cargo build --workspace`?
+- [ ] Workspace tests `cargo test --workspace`?
```

Comment:
That might be to heavy for some external contributors' laptops to handle ;)

Reply:
Good point :+1:

Reply:
Because the turnaround time is significantly faster doing it locally iff your local machine can handle it :)

Reply:
Added additional indentation